components/dashboard/src/components/WorkspaceLogs.tsx

```diff
@@ -24,6 +24,7 @@ export interface WorkspaceLogsProps {
     logsEmitter: EventEmitter;
     errorMessage?: string;
     classes?: string;
+    xtermClasses?: string;
```

Comment:
nit: It's a bit weird that we override classes here instead of merging. Do you think we can use our `cn` utility function to make it consistent with our other components? To keep things straight forward, we could rename `classes` to `className` to also keep it consistent.